packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Buttons/__tests__/buttons.spec.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { createTestInstance } from '@magento/peregrine';
+import Buttons from '../Buttons';
+jest.mock('../../../../../../classify');
+
+test('renders a Buttons component', () => {
+    const component = createTestInstance(<Buttons />);
+
+    expect(component.toJSON()).toMatchSnapshot();
+});
+test('renders a Buttons component with stacked appearance', () => {
+    const buttonsProps = {
+        appearance: 'stacked',
+        border: 'solid',
+        borderColor: 'rgb(0, 0, 0)',
+        borderRadius: '10px',
+        borderWidth: '1px',
+        cssClasses: ['test-class'],
+        marginBottom: '10px',
+        marginLeft: '10px',
+        marginRight: '10px',
+        marginTop: '10px',
+        paddingBottom: '10px',
+        paddingLeft: '10px',
+        paddingRight: '10px',
+        paddingTop: '10px',
+        textAlign: 'center'
+    };
+    const component = createTestInstance(<Buttons {...buttonsProps} />);
+
+    expect(component.toJSON()).toMatchSnapshot();
+});
+
+test('renders a Buttons component with inline appearance and same width', () => {
```

Comment:
This test isn't actually verifying the buttons are the same size, we'll want to ensure the callbacks are ran correctly as part of this.